src/index.ts

```diff
@@ -60,11 +60,11 @@ extendConfig((config: HardhatConfig, userConfig: Readonly<HardhatUserConfig>) =>
 // add url to alpha network
 extendConfig((config: HardhatConfig) => {
```

Comment:
Maybe it makes sense to add a default verification URL if the user does not provide one, or at least return an error.